convert/tests/test_vertex/test_text_wrapper.py

```diff
@@ -0,0 +1,84 @@
+import collections
+import string
+
+from src.vertex_to_bd.badgerdoc_format.plain_text_converter import TextWrapper
+
+
+def test_pop_beginning_whitespaces_text_begin_with_whitespaces():
+    tw = TextWrapper(line_length=50)
+    text = collections.deque(string.whitespace + "text" + "  ")
+
+    result = tw.pop_beginning_whitespaces(text)
+
+    assert result == list(string.whitespace)
+
+
+def test_pop_beginning_whitespaces_not_whitespaces():
+    tw = TextWrapper(line_length=50)
+    text = collections.deque("text" + "  ")
+
+    result = tw.pop_beginning_whitespaces(text)
+
+    assert result == []
+
+
+def test_pop_next_word_begin_with_whitespaces():
+    tw = TextWrapper(line_length=50)
+    text = collections.deque(" " + "text")
+
+    result = tw.pop_next_word(text)
+
+    assert result == []
+
+
+def test_pop_next_word_text_contains_several_verbs():
+    tw = TextWrapper(line_length=50)
+    text = collections.deque("word1 word2 word3")
+
+    result = tw.pop_next_word(text)
+
+    assert result == ["w", "o", "r", "d", "1"]
+
+
+def test_wrap_short_text():
+    tw = TextWrapper(line_length=40)
+    text = "word1 word2 word3 word4 word5"
+
+    result = tw.wrap(text)
+
+    assert result == [text]
+
+
+def test_wrap_text_started_with_spaces():
+    tw = TextWrapper(line_length=20)
```

Comment:
please name the magic number

Reply:
I've removed the numbers from expected result output